lib/runner/index.js

```diff
@@ -39,13 +39,21 @@ _.assign(Runner.prototype, {
      * @param {Function} callback
      */
     run: function (collection, options, callback) {
-        var runOptions = _.merge(_.omit(options, ['environment', 'globals', 'data']), this.options.run);
+        var runnableItems,
+            runOptions = _.merge(_.omit(options, ['environment', 'globals', 'data']), this.options.run);
 
         callback = backpack.normalise(callback);
         !_.isObject(options) && (options = {});
 
+        runnableItems = Runner.extractRunnableItems(collection, options.entrypoint);
+
+        // Bail out only if: abortOnError is set and the returned entrypoint is invalid
+        if (options.abortOnError && !runnableItems.entrypoint) {
```

Comment:
`extractRunnableItems` returns array, then what is this check `!runnableItems.entrypoint`?